internals/secrethub/mkdir.go

```diff
@@ -33,6 +34,7 @@ func NewMkDirCommand(io ui.IO, newClient newClientFunc) *MkDirCommand {
 func (cmd *MkDirCommand) Register(r command.Registerer) {
 	clause := r.Command("mkdir", "Create a new directory.")
 	clause.Arg("dir-path", "The path to the directory").Required().PlaceHolder(dirPathPlaceHolder).SetValue(&cmd.path)
+	clause.Flag("parents", "Create parent directories if needed. Does not error when directories already exist.").Short('p').BoolVar(&cmd.parents)
```

Comment:
Turns out the `p` shorthand is already in use for `--credential-passphrase`.

Reply:
Ugh. I don't like that. Do we even need a shorthand for that? It's dubious flag anyway.

Reply:
Yes, I agree.

How shall we migrate? Are we oke with removing the shorthand or do we want to keep it for backwards compatibility? I don't expect (a lot of) usage and the migration path is really easy (just use `--credential-passphrase` instead), but it can be pretty destructive.

Reply:
We can merge it for now without the shorthand, hide the `-p` for credential passphrase and create a deprecation notice for the current short-hand. Then we can add the short-hand to this command later.

Reply:
See #226 for hiding the `-p` shorthand of `--credential-passphrase`